subsys/nrf_security/src/drivers/cracen/psa_driver.Kconfig

```diff
@@ -1725,6 +1725,7 @@ config PSA_NEED_CRACEN_CTR_DRBG_DRIVER
 	default y
 	select PSA_ACCEL_GENERATE_RANDOM
 	# Is needed for AES counter measurements
+	depends on PSA_WANT_GENERATE_RANDOM
```

Comment:
Move the comment above below this line?

Reply:
I don't get it sorry, the line that you are pointing is exactly after the comment.

Reply:
My wording might've been a bit confusing. I meant, `move the comment that is right now above this line below it?` because the comment is about AES and counter measures and it feels like it was addressing the line below unless I am missing something?

Reply:
Hm, the aes countermeasure requires the PSA_WANT_GENERATE_RANDOM. So isn't this logical?

Reply:
Ah okay. Fine then. It's just that this comment used to be addressed to the `depends on` statement.